src/components/Sum.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Sum({ a, b }) {
+  return (
+    <p>
+      Sum of
+      {a}
+      {' '}
+      and
+      {b}
+      {' '}
+      is
+      {a + b}
```

Comment:
There is much easier way to render this text:
```suggestion
  {`Sum of ${a} and ${b} is ${a + b}`}
```

Reply:
It is automatically changed by linter and I don't know how to fix it